tests/test.display.js

```diff
@@ -407,7 +402,7 @@ describe('Display/Canvas Helper', function () {
         });
 
         it('should wait until an image is loaded to attempt to draw it and the rest of the queue', function () {
-            var img = { complete: false };
+            var img = { complete: false, addEventListener: sinon.spy() }
```

Comment:
do we want to test this with an actual image?  IIRC, sinon has ways to test asynchronous tests, and I'd rather test the actual code path here.